test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go

```diff
@@ -23,6 +23,13 @@ package resource_test
 // Tests in this file should only use cloud resources if absolutely necessary.
 //
 // Tests in this file should be kept *roughly* in sync with recipe_bicep_test and any other drivers.
+//
+// Note: To test the Terraform container mounting feature (pre-mounted binaries), you would need to:
```

Comment:
These tests do not run for me using a local k3d cluster with Radius installed using `rad install kubernetes --terraform-container`, but maybe I'm missing some setup.